Library/Formula/ghi.rb

```diff
@@ -7,6 +7,8 @@ class Ghi < Formula
 
   bottle :unneeded
 
+  depends_on "pygments.rb" => :ruby
```

Comment:
That still won’t work (just checked in a VM), because HB doesn’t actually run the `gem install` command with its own ruby. It will just tell users to `gem install pygments.rb`, and after they do it will be installed in their managed ruby, leaving this in the same situation as before.